command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
@wking said:  `So I don't see a need (beyond the better
compliance-testing language [1]) to underline the granularity for this
particular portion of the spec.`   

It's a matter of defining terms and then being consistent with their use.  Runtime is a broad term.  Runtime-CLI is more specific.